src/dotnet/CarbonAware.WebApi.Tests/unitTests/CarbonAwareControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+namespace CarbonAware.WepApi.UnitTests;
+
+using System.Collections.Generic;
+using CarbonAware.Model;
+using CarbonAware.WebApi.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+/// <summary>
+/// Testing Web API Controller
```

Comment:
nit: I think you can write a better XML doc summary than this :) It's very clear that's what this is but maybe you can elaborate on what the strategy is for this test suite?